src/models/StateType.ts

```diff
@@ -16,6 +16,8 @@ export function isStateType(value: unknown): value is StateType {
   return typeof value === 'string' && stateType.includes(value as StateType)
 }
 
+export const scheduledStateType = ['scheduled', 'pending']
```

Comment:
should there be a `isScheduledStateType` utility to go with this?

Reply:
Nope, this shouldn't have been added at all! Removed